.travis.yml

```diff
@@ -36,10 +36,10 @@ jobs:
         - oc rollout status dc/postgresql
         - REVISION=$(oc rollout latest dc/postgresql -o revision)
         - oc rollout status dc/postgresql --revision=$REVISION
-        - oc rollout status dc/cloudigrade
+        - oc rollout status dc/cloudigrade-api
         - REVISION=$(oc rollout latest dc/cloudigrade -o revision)
```

Comment:
You missed this guy. Also needs to be `dc/cloudigrade-api`.

Reply:
good catch